src/components/AppDropdownSelect.tsx

```diff
@@ -48,14 +55,16 @@ export const AppDropdownSelect = ({
         </View>
       </TouchableWithoutFeedback>
       {isExpanded ? (
-        <View style={styles.expanded}>
+        <View style={{ ...styles.expanded, ...dropdownContainerStyle }}>
```

Comment:
```suggestion
        <View style={[styles.expanded, dropdownContainerStyle]}>
```
i w innych miejscach